content/quickstart/node/tracing.md

```diff
@@ -75,8 +75,8 @@ stdin.addListener("data", readEvaluateProcessLine);
 ## Getting Started
 The Repl application takes input from users, converts any lower-case letters into upper-case letters, and echoes the result back to the user, for example:
 ```bash
-> foo
```

Comment:
Would you mind actually using the suggestion for `process.stdout.write`? That way we can achieve the consistency that we'd want with other languages.